website/docs/r/synapse_sql_pool.html.markdown

```diff
@@ -59,6 +59,8 @@ The following arguments are supported:
 
 * `sku_name` - (Required) Specifies the SKU Name for this Synapse Sql Pool. Possible values are `DW100c`, `DW200c`, `DW300c`, `DW400c`, `DW500c`, `DW1000c`, `DW1500c`, `DW2000c`, `DW2500c`, `DW3000c`, `DW5000c`, `DW6000c`, `DW7500c`, `DW10000c`, `DW15000c` or `DW30000c`.
 
+* `data_encrypted` - (Optional) Is transparent data encryption enabled? Defaults to `false`.
```

Comment:
since the rest of this file is sorted Required -> Optional and then alphabetically, can we move this down to match that?